articles/techniques/index.md

```diff
@@ -12,26 +12,38 @@ uid: microsoft.quantum.techniques.intro
 
 # Quantum Development Techniques
 
-![Quantum](~/media/mobius_strip_preview.png)
-
-This section details the core concepts used to create quantum programs in Q#, and to interact with those programs from classical .NET applications.
-This section assumes some knowledge of quantum computing concepts like those described in [Quantum computing concepts](xref:microsoft.quantum.concepts.intro).
-
-
-
-
-
-
-
+This section of our documentation details the core concepts used to create quantum programs in Q#, and to interact with those programs from classical .NET applications.
+We assume *some* knowledge of quantum computing concepts, like those described in [Quantum computing concepts](xref:microsoft.quantum.concepts.intro), but you needn't be an expert in quantum computing to get a lot from these sections.
 
+Their contents are as follows.
 
+- [Q# program overview](xref:microsoft.quantum.techniques.file-structure) provides an overview of the purpose and functionality of the Q# programming language. 
+	In particular, it clarifies how Q# is *not* a language for merely simulating quantum mechanics---though that functionality is of course provided by our full state simulator. 
+	Rather, Q# was designed with an eye on the future, and its programs describe how a classical control computer *interacts* with qubits. 
 
+- [Operations and functions](xref:microsoft.quantum.techniques.opsandfunctions) details the two callable types of the Q# language: *operations*, which include action on qubits and quantum systems; and *functions*, which strictly work with classical information. 
+	Without both classical and quantum information working in tandem, quantum computing would remain out of reach. 
+	This section describes how to define and use these callables within the control flow of a Q# program.
 
+- [Local variables](xref:microsoft.quantum.techniques.local-variables) describes the role of variables within Q# programs and how to leverage them effectively. 
+	In particular, you will learn the difference between immutable/mutable variables and how to assign/re-assign them.
 
+- [Working with qubits](xref:microsoft.quantum.techniques.qubits) describes the features of Q# that you can use to address individual qubits and systems of qubits. 
+	Specifically, that entails their allocation, or blocking off a certain number of qubits for an operation or program to use; performing operations on them; and ultimately their measurement. 
```

Comment:
I can't quite put my finger on it, sorry, but "blocking off" feels a bit awkward here?

Reply:
@cgranade I know what you mean. Looking back on it, I feel like it's not totally necessary to explain what "allocation" means here. Perhaps just this?
```suggestion
	Specifically, that entails allocating qubits, performing operations on them, and ultimately their measurement. 
```

Reply:
I think that works pretty well, yeah.